packages/supermassive/src/executeWithoutSchema.ts

```diff
@@ -764,6 +764,7 @@ export function buildResolveInfo(
     rootValue: exeContext.rootValue,
     operation: exeContext.operation,
     variableValues: exeContext.variableValues,
+    fieldNodes: fieldGroup,
```

Comment:
Let's copy what tehy do in graphql-js, this isn't actually compatible it seems.

Reply:
It is, graphql-js just did another round of changes in 17-alpha, so the code they have today in https://github.com/graphql/graphql-js/blob/8d7c8fccf5a9846a50785de04abda58a7eb13fc0/src/execution/execute.ts#L699 is not what we have in supermassive v3. The test in this PR confirms that it is compatible, resolving.